defaults/main.yml

```diff
@@ -19,3 +19,6 @@ nodejs_npm_global_packages: []
 #    version: 0.9.3
 #  # Install the latest stable release of a package.
 #  - name: node-sass
+
+# The path of a package.json
+nodejs_package_json_path:
```

Comment:
In this definition, can you set it to an empty string so it's more obvious a string should be entered?

```yaml
nodejs_package_json_path: ""
```

Also, can you add the variable, then a sentence or two describing what it does, to the README.md?